valhalla/userrequests/state_changes.py

```diff
@@ -106,7 +106,7 @@ def debit_ipp_time(ur):
         return
     try:
         time_allocations = ur.timeallocations
```

Comment:
does line ~84, validate_ipp function need changes too?

Reply:
Yea, it did.

Reply:
Where are the unit tests for this function? Can't find any. I would expect them to fail.